docs/cypress-testing-library/intro.mdx

```diff
@@ -58,9 +58,9 @@ cy.findByRole('button', {name: /Button Text/i}).should('exist')
 cy.findByRole('button', {name: /Non-existing Button Text/i}).should('not.exist')
 cy.findByLabelText(/Label text/i, {timeout: 7000}).should('exist')
 
-// findAllByText _inside_ a form element
+// findByText _inside_ a form element
 cy.get('form')
-  .findByText('button', {name: /Button Text/i})
+  .findByText(/Button Text/i)
```

Comment:
Have you tried using `getByRole`?
Reason: `getByRole` is the recommended selector to query elements, https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-byrole-most-of-the-time

```suggestion
  .findByRole('button', {name: /Button Text/i})
```

Reply:
Thanks for the suggestion of using `findByRole`. This is what I would generally use in this situation as well, but in this PR I was not trying to make any judgement on the correctness of the selector but rather to use the idea that was already there appropriately.

I was assuming that, since the example already uses `findByRole` multiple times (you can see them if you expand the diff), the example author wanted to show a different API and used `findByText`. I guess it could alternatively be seen as wanting to use `findByRole` again because the arguments match that.

I am fine with either version and just want the example to be valid. Since it seems better for the example to follow bet practices I will switch to `findByRole`.